payment/src/main/java/com/kyriba/conference/payment/domain/PaymentEntity.java

```diff
@@ -0,0 +1,76 @@
+package com.kyriba.conference.payment.domain;
+
+import com.kyriba.conference.payment.api.dto.PaymentDto;
+import com.kyriba.conference.payment.api.dto.Receipt;
+import com.kyriba.conference.payment.api.dto.TicketDto;
+import com.kyriba.conference.payment.domain.dto.Amount;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.util.Currency;
+
+/**
+ * @author Igor Lizura
+ */
+@Entity
+@Table(name = "payment")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class PaymentEntity {
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_identity_id")
+    @SequenceGenerator(name = "seq_identity_id", sequenceName = "seq_identity_id", allocationSize = 1)
+    @Id
+    @Column(name = "id", updatable = false, nullable = false)
```

Comment:
redundant?

Reply:
redundant what?

Reply:
annotation in general, but also I think 'updatable = false, nullable = false' since this is Id

Reply:
Column annotation is useless here.

Reply:
fixed